### ISSUE TITLE
server.py, line 282: NotImplementedError

### ISSUE THREAD

**Issue by p-hash** [aio-libs/aiohttp#1703]
## Long story short

I get this error:
```
Error handling request
Traceback (most recent call last):
  File "/home/paul/mpb-v2/env/lib/python3.4/site-packages/aiohttp/server.py", line 282, in start
    self.transport.write_eof()
  File "/usr/lib64/python3.4/asyncio/transports.py", line 111, in write_eof
    raise NotImplementedError
NotImplementedError
```
It was mentioned before in #1537 

## Steps to reproduce

Thanks to @freekvw we have the code that reproduces the issue.
Error happens when you upload a file larger than 50KiB
```
from aiohttp import web
import ssl
import asyncio

@asyncio.coroutine
def handle(request):
    text = """<html><body>
<form action="/upload" method="post" accept-charset="utf-8"
      enctype="multipart/form-data">
    <input name="file" type="file" value=""/><br />
    <input type="submit" value="submit"/>
</form></body></html>"""
    return web.Response(text=text, content_type='text/html')

@asyncio.coroutine
def handleUpload(request):
    return web.json_response({})

app = web.Application()
app.router.add_get('/', handle)
app.router.add_post('/upload', handleUpload)

import proxybot_config as config
context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
context.load_cert_chain(config.SSL_CERT, config.SSL_PKEY)

web.run_app(app,
  host=config.LISTEN, 
  port=config.PORT, 
  ssl_context=context)
```

## Your environment

OS: Fedora 23
Python 3.4.3
aiohttp 1.3.3

**Comment by fafhrd91** [comment]
@p-hash could you try this repo https://github.com/fafhrd91/aiohttp

**Comment by ttyridal** [comment]
Found this as I'm experiencing the same issue.  Ubuntu 16.10, Python 3.5.2, aiohttp 1.3.3

@fafhrd91 Your repo seems to have fixed the issue for me at least..

That's while noting that I got
```
x86_64-linux-gnu-gcc: error: aiohttp/_websocket.c: No such file or directory
x86_64-linux-gnu-gcc: fatal error: no input files
compilation terminated.
************************************************************
Cannot compile C accelerator module, use pure python version
```
during setup.py

**Comment by fafhrd91** [comment]
do you have cython installed?

**Comment by ttyridal** [comment]
good point.. still working and now, no build errors.

**Comment by fafhrd91** [comment]
fixed in master and in 1.3 branch
will release 1.3.4 soon
